platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/network/NetworkMetrics.java

```diff
@@ -101,13 +102,20 @@ public NetworkMetrics(final Metrics metrics, final NodeId selfId, final int addr
         avgPing = metrics.getOrCreate(AVG_PING_CONFIG);
         bytesPerSecondSent = metrics.getOrCreate(BYTES_PER_SECOND_SENT_CONFIG);
         avgConnsCreated = metrics.getOrCreate(AVG_CONNS_CREATED_CONFIG);
+        disconnectFrequency = IntStream.range(0, addressBookSize)
```

Comment:
Am I reading correctly, that for this new metric `nodeId` is presumed to be equal to address book index? I don't think that's a conflation we have committed/will commit to